build/csharpTranspiler.ts

```diff
@@ -23,6 +23,15 @@ const exchangeIds = exchanges.ids
 
 let __dirname = new URL('.', import.meta.url).pathname;
 
+function overwriteSafe (path, content) {
```

Comment:
I don't like this function, first it's not clear what safe means here, second matches any exception blindly and creates the folder even if the exception is unrelated